application/helpers/common_helper.php

```diff
@@ -5087,3 +5087,23 @@ function recursive_preg_replace($pattern, $replacement, $subject, $limit = -1, &
     }
     return $result;
 }
+
+/**
+ * function to calculate the standard deviation of an array
```

Comment:
Maybe rephrase as

"Returns the standard deviation of $numbers" or "Calculates the standard deviation of $numbers"